src/app/loadout-drawer/LoadoutDrawerContents.tsx

```diff
@@ -160,6 +167,9 @@ export default function LoadoutDrawerContents({
         <a onClick={() => onOpenModPicker()} className="dim-button loadout-add">
           <AppIcon icon={addIcon} /> {t('Loadouts.ArmorMods')}
         </a>
+        <a onClick={() => onOpenFashionDrawer()} className="dim-button loadout-add">
+          <AppIcon icon={faTshirt} /> {t('Loadouts.Fashion')}
+        </a>
```

Comment:
This appears to be the only way to look at the fashion from the loadout drawer. It'd be nice if the selected fashion was reflected in the contents below so that it's obvious what's in the loadout, what could be changed and what would be saved when clicking the save button.

Reply:
Yeah - it's a good callout but I'm hoping to not have to build an interim edit-display view.